src/igvfd/schemas/construct_library.json

```diff
@@ -0,0 +1,353 @@
+                    "not": {
+                        "required": [
+                            "upper_bound_coverage_range",
+                            "lower_bound_coverage_range"
+                        ]
+                    },
+                    "comment": "Specification of average_guide_coverage is mutually exclusive with specification of upper_ and lower_bound_coverage_range."
+                },
+                "lower_bound_coverage_range": {
+                    "not": {
+                        "required": [
+                            "average_guide_coverage"
+                        ]
+                    },
+                    "required": [
+                        "upper_bound_coverage_range"
+                    ],
+                    "comment": "Specification of lower_bound_coverage_range is mutually exclusive with specification of average_guide_coverage."
+                },
+                "upper_bound_coverage_range": {
+                    "not": {
+                        "required": [
+                            "average_guide_coverage"
+                        ]
+                    },
+                    "required": [
+                        "lower_bound_coverage_range"
+                    ],
+                    "comment": "Specification of upper_bound_coverage_range is mutually exclusive with specification of average_guide_coverage."
+                }
+            },
+            "properties": {
+                "guide_type": {
+                    "title": "Guide Type",
+                    "description": "The design of guides used in a CRISPR library, paired-guide (pgRNA) or single-guide (sgRNA).",
+                    "type": "string",
+                    "enum": [
+                        "sgRNA",
+                        "pgRNA"
+                    ]
+                },
+                "tiling_modality": {
+                    "title": "Tiling Modality",
+                    "description": "The tiling modality of guides across elements or loci in a CRISPR library.",
+                    "comment": "Peak tiling = all possible gRNA in a selection of peaks. Full tiling = all possible gRNA across an entire locus, including regulatory elements and non regulatory elements. Sparse peaks = a small set of gRNA against a selection of peaks.",
+                    "type": "string",
+                    "enum": [
+                        "peak tiling",
+                        "full tiling",
+                        "sparse peaks"
+                    ]
+                },
+                "average_guide_coverage": {
+                    "title": "Average Guide Coverage",
+                    "description": "The average number of guides targeting each element of interest in the library.",
+                    "type": "number",
+                    "minimum": 0
+                },
+                "lower_bound_guide_coverage": {
+                    "title": "Lower Bound Insert Size",
```

Comment:
Possible copy paste error, I think this title is matching the reporter_library_details subproperties and probably needs to be updated.